habitat-lab/habitat/tasks/ovmm/utils.py

```diff
@@ -0,0 +1,30 @@
+import habitat_sim
+
+
+def find_closest_goal_index_within_distance(
```

Comment:
Curious if you know how expensive this function is? It looks quite expensive to me if you are doing it multiple times in a single episode

Reply:
I think we can avoid calling this in OvmmRotDistToGoal, if we are using the coverage-based metrics- that way we would use this only once per episode. I think we should use only one at a time. I will make the change.

Reply:
Since this is only to make things efficient, will change this later.